synapseclient/extensions/curator/schema_generation.py

```diff
@@ -5295,14 +5416,20 @@ def get_component_json_schema(
         Raises:
             May raise errors if the component is not found in the data model graph.
         """
-
+        metadata_model = MetadataModel(
```

Comment:
This is part of the original functionality in schematic: https://github.com/Sage-Bionetworks/schematic/blob/develop/schematic/schemas/json_schema_component_generator.py#L245